rollup.config.js

```diff
@@ -7,6 +7,7 @@ import pkg from './package.json'
 // These plugins are used for all builds
 const sharedPlugins = [
   replace({
+    preventAssignment: false,
```

Comment:
Added this because rollup gives a deprecation warning about the default value of this flag changing in the next major, so I pinned this to the current default pre-emptively.